site/src/Librecores/ProjectRepoBundle/Entity/Organization.php

```diff
@@ -59,37 +60,43 @@ class Organization
 
     /**
      * @var ArrayCollection
+     *
      * @ORM\OneToMany(targetEntity="Project", mappedBy="parentOrganization")
      * @ORM\JoinColumn(name="projectId", referencedColumnName="id")
      */
     private $projects;
 
     /**
-     * @var User
-     * @ORM\ManyToOne(targetEntity="User", inversedBy="organizationsOwner")
-     * @ORM\JoinColumn(name="ownerId", referencedColumnName="id")
-     **/
-    protected $owner;
-
-    /**
      * @var ArrayCollection
-     * @ORM\ManyToMany(targetEntity="User", inversedBy="organizationsMember")
-     * @ORM\JoinTable(name="OrganizationMembers",
-     *     joinColumns={@ORM\JoinColumn(name="organizationId", referencedColumnName="id")},
-     *     inverseJoinColumns={@ORM\JoinColumn(name="memberId", referencedColumnName="id")}
-     *     )
+     *
+     * @ORM\OneToMany(targetEntity="OrganizationMember", mappedBy="organization", cascade={"remove"}))
      **/
     protected $members;
 
     /**
-     * @var ArrayCollection
-     * @ORM\ManyToMany(targetEntity="User", inversedBy="organizationsRequest")
-     * @ORM\JoinTable(name="OrganizationRequests",
-     *     joinColumns={@ORM\JoinColumn(name="organizationId", referencedColumnName="id")},
-     *     inverseJoinColumns={@ORM\JoinColumn(name="requestId", referencedColumnName="id")}
-     *     )
-     **/
-    protected $requests;
+     * @var User
+     *
+     * @ORM\ManyToOne(targetEntity="User", inversedBy="organizationsCreated")
+     * @ORM\JoinColumn(name="creatorId", referencedColumnName="id")
+     */
+    private $creator;
+
+    // metadata
+    /**
+     * When was this organization created?
+     *
+     * @var \DateTime
+     * @ORM\Column(type="datetime", nullable=true)
```

Comment:
It was false until yesterday and I agree with what you say. However I was forced to change it when I encountered an exception from mysql (can't recall the exact situation) that I didn't have time to resolve. I'll try to investigate further and see how it goes so that we can set it back to false.